website/docs/r/user_assigned_identity.html.markdown

```diff
@@ -15,6 +15,11 @@ Manages a User Assigned Identity.
 ## Example Usage
 
 ```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
```

Comment:
the documentation for this resource is auto-generated / comes from Pandora - so we'll need to fix this there fwiw, else this'll get regenerated/reverted